tests/acceptance/qunit-dom-test.js

```diff
@@ -18,6 +18,7 @@ module('Acceptance | qunit-dom', function(hooks) {
     assert.dom('#title').exists();
     assert.dom('#subtitle').doesNotExist();
     assert.dom('#title').hasText('Welcome to Ember');
+    assert.dom('#title').hasTagName('h2');
```

Comment:
FWIW we don't have to add all the assertions here. this is just meant as a smoke test that the qunit-dom integration works correctly. the assertions itself are tested enough by the Jest test suite.